CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+### Fixed
```

Comment:
Should ideally have the unrelease/added/changed/fixed/removed sections like in https://raw.githubusercontent.com/oxyplot/oxyplot/StemSeriesActualMarker/CHANGELOG.md , but we can do that separately